Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12
```

Comment:
Node 14 is the new LTS version, please use it instead.

Reply:
Added